db/migrate/20241203151206_add_points_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPointsToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :points, :integer
```

Comment:
a w sumie jak to wyglada z punktami? Czy to nie powinna byc relacja wiele do wiele miedzy uzytkownikiem a commentem? Zeby zapisywac ktory uzytkownik juz polubil i zeby wiecej razy nie mogl?